ext/standard/math.c

```diff
@@ -276,8 +298,8 @@ PHP_FUNCTION(abs)
 
 /* {{{ Returns the next highest integer value of the number */
 PHP_FUNCTION(ceil)
-{
-	zval *value;
+		{
+				zval * value;
```

Comment:
This looks like some autoformatter mangled the file, making it hard to review the actual changes.

Reply:
Thank you. It looks ugly. I removed this formatting.